src/components/ClusterManager.js

```diff
@@ -273,13 +255,14 @@ export default class ClusterManager extends PureComponent {
             shape: 'play',
             onClick: () => this.setState({ createModalDrawerOpen: true }),
             disabled: busy || !canCompute,
-            tooltip: canCompute ? 'Create notebook runtime' : noCompute,
-            'aria-label': 'Create notebook runtime'
+            tooltip: canCompute ? 'Create cloud environment' : noCompute,
+            'aria-label': 'Create cloud environment'
```

Comment:
TOL: Should this aria-label have the same ternary as above? (Existing issue if so, but may be worth thinking about?)

Reply:
this is also a thing in the above lines 241, 230, and 222. I haven't tested yet but i imagine the ternary should be used in the aria-label as you've suggested. Will look in the UI and see how it feels and report back

Reply:
I wonder if there's an expectation that `aria-label` should be stable over time so people don't get lost in an ever-changing landscape.

Reply:
canCompute is unlikely to change on folks though, right? Although I guess if I am using a screen reader and bouncing from one workspace to another, I might expect it to look the same.

Reply:
That is exactly the thought process I went thorough as I was adding my comment. :)

Reply:
I think this is a reasonable UX as-is. In fact, the accessible experience may be better than the visual one in this case. There's an `aria-label` that describes what the button does, and on focus/hover, we include an `aria-describedby` pointing to the tooltip, which describes the reason it's disabled (if applicable).